### PR TITLE
Dịch "linear-regression_vn" - Phần 5

### LINKED ISSUE #549: Dịch "linear-regression_vn" - Phần 5
Phần cần dịch: https://github.com/aivivn/d2l-vn/blame/master/chapter_linear-networks/linear-regression_vn.md#L165-L221

### PR DESCRIPTION
**Pull request by thduynguyen**
## Pull Request Checklist

**Đánh dấu tick (sau khi "Create Pull Request") vào các mục dưới đây**

* [x] Pull Request này tương ứng với issue nào? Trả lời số issue sau dấu `#` (không có dấu cách): Close #549 

* [x] Bản dịch này có bám sát **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)** không? Nếu một từ/cụm chưa có trong Bảng thuật ngữ, hãy tạo một Pull Request riêng đề xuất cách dịch từ/cụm đó.

* [x] Format của bản gốc có được giữ nguyên không?
    * Các phần in nghiêng, in đậm.
    * Tiêu đề (số lượng dấu `#` đầu dòng).
    * Bảng biểu, chú thích cho bảng (dòng phía trên mỗi bảng bắt đầu bằng dấu `:`).
    * Dịch các từ trong hình vẽ nếu cần. Các dòng có hình có dạng: `caption`.
    * Dịch các chú thích hình vẽ (thay các cụm `*dịch chú thích ảnh phía trên*` bằng bản dịch tiếng Việt).
    * Không dịch các phần code (nằm giữa hai dấu `````).
    * Copy các công thức toán từ bản gốc (các đoạn có `$`).
    * Giữ các dòng gán nhãn (bắt đầu với `:label:`).
    * Không tự thêm bớt các dòng trắng.
     
* [x] Trong một đoạn văn, mỗi câu văn đã được viết trong một dòng, giữa các dòng không có dòng trắng. Markdown sẽ coi những dòng liền nhau không có dòng trắng là một đoạn văn. Việc này giúp công đoạn review được thuận tiện hơn.

* [x] Tất cả chữ trong các hình vẽ đã được dịch. Nếu chưa, vui lòng xem thêm [hướng dẫn hoặc yêu cầu dịch hình](../img/README.md).

* [x] Điền tên của bạn vào mục "Những người thực hiện" ở cuối file.

* [ ] **Sau khi được approve**, điền tên những reviewers mà bạn thấy có nhiều đóng góp cho Pull Request này. Xem danh sách Contributors **[tại đây](../docs/contributors_info.md/)**.

## Với reviewer

1. Chỉ review nếu Pull Request này đã sẵn sàng cho việc review.

2. **Bạn có thể đề xuất thay đổi cách dịch mỗi dòng trực tiếp như trong hình dưới đây:**
img

### CODE REVIEW

File: chapter_linear-networks/linear-regression_vn.md [hunk 1]
```diff
@@ -186,7 +186,10 @@ The most popular loss function in regression problems is the sum of squared erro
 When our prediction for some example $i$ is $\hat{y}^{(i)}$ and the corresponding true label is $y^{(i)}$, the squared error is given by:
 -->
 
-*dịch đoạn phía trên*
+Trước khi suy nghĩ làm thế nào để *khớp* mô hình vào với dữ liệu, ta cần phải xác định một cách đo cho *mức độ khớp*.
```
Comment (rootonchair) [inline chapter_linear-networks/linear-regression_vn.md]:
```suggestion
Trước khi suy nghĩ làm thế nào để *khớp* mô hình vào với dữ liệu, ta cần phải xác định một cách để đo *mức độ khớp*.
```

File: chapter_linear-networks/linear-regression_vn.md [hunk 2]
```diff
@@ -186,7 +186,10 @@ The most popular loss function in regression problems is the sum of squared erro
...
+Hàm mất mát định lượng khoảng cách giữa giá trị *thực* và giá trị *dự đoán* của mục tiêu.
```
Comment (rootonchair) [inline chapter_linear-networks/linear-regression_vn.md]:
```suggestion
*Hàm mất mát* định lượng khoảng cách giữa giá trị *thực* và giá trị *dự đoán* của mục tiêu.
```

File: chapter_linear-networks/linear-regression_vn.md [hunk 3]
```diff
@@ -186,7 +186,10 @@ The most popular loss function in regression problems is the sum of squared erro
 When our prediction for some example $i$ is $\hat{y}^{(i)}$ and the corresponding true label is $y^{(i)}$, the squared error is given by:
```
Comment (rootonchair) [inline chapter_linear-networks/linear-regression_vn.md]: Bạn dịch thiếu chỗ này rồi nhé

File: chapter_linear-networks/linear-regression_vn.md [hunk 4]
```diff
@@ -196,29 +199,32 @@ Since the training dataset is given to us, and thus out of our control, the empi
...
+Để đo chất lượng của mô hình trên toàn bộ tập dữ liệu, ta chỉ đơn giản lấy trung bình (hay tương đương là lấy tổng) của các độ mất mát trên tập huấn luyện.
 
 $$L(\mathbf{w}, b) =\frac{1}{n}\sum_{i=1}^n l^{(i)}(\mathbf{w}, b) =\frac{1}{n} \sum_{i=1}^n \frac{1}{2}\left(\mathbf{w}^\top \mathbf{x}^{(i)} + b - y^{(i)}\right)^2.$$
 
 <!--
 When training the model, we want to find parameters ($\mathbf{w}^*, b^*$) that minimize the total loss across all training samples:
 -->
 
-*dịch đoạn phía trên*
+Khi huấn luyện mô hình, chúng ta tìm các tham số ($\mathbf{w}^*, b^*$) để cực tiểu hóa tổng độ mất mát trên toàn bộ tất cả các mẫu huấn luyện:
```
Comment (tiepvupsu) [inline chapter_linear-networks/linear-regression_vn.md]:
```suggestion
Khi huấn luyện mô hình, chúng ta tìm các tham số ($\mathbf{w}^*, b^*$) để tối thiểu hóa tổng độ mất mát trên toàn bộ tất cả các mẫu huấn luyện:
```

`tối thiểu` mang nghĩa `global`, còn `cực tiểu` mang nghĩa `local`.
